.appveyor.yml

```diff
@@ -7,7 +7,7 @@ skip_tags: true
 # Branches to clone
 branches:
   only:
-    - master
+    - decompiler-refactoring
```

Comment:
You can probably add decompiler-refactoring to a list instead of replacing master. That way it will be easier to cleanup once everything is done.

Reply:
Changed